README.md

```diff
@@ -2,78 +2,91 @@
-- [Current Bidders](https://docs.prebid.org/dev-docs/pbs-bidders.html)
+Refer to the [configuration guide](docs/developers/configuration.md) for additional information and a list of available configuration options.
 
-Please consider [registering your Prebid Server](https://docs.prebid.org/prebid-server/hosting/pbs-hosting.html#optional-registration) to get on the mailing list for updates, etc.
+## Hosting Prebid Server
+> [!NOTE]
+> Please consider [registering as a Prebid Server host](https://docs.prebid.org/prebid-server/hosting/pbs-hosting.html#optional-registration) to join the mailing list for updates and feedback.
 
-## Installation
+The quickest way to host Prebid Server is to deploy our [official Docker image](https://hub.docker.com/r/prebid/prebid-server). If you're hosting the container with Kubernetes, you can configure Prebid Server with environment variables [using a pod file](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/) or [using a config map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables). Alternatively, you can use a configuration file [embedded in a config map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap) which Prebid Server will read from at the path `/etc/config`.
 
-First install [Go](https://golang.org/doc/install) version 1.19 or newer.
+For deploying a fork, you can create a custom Docker container using the command:
+``` bash
+docker build -t prebid-server .
+```
+or compile a standalone binary using the command:
+``` bash
+go build .
+```
+Ensure that you deploy the `/static` directory, as Prebid Server requires those files at startup.
 
-Note that prebid-server is using [Go modules](https://blog.golang.org/using-go-modules).
-We officially support the most recent two major versions of the Go runtime. However, if you'd like to use a version <1.13 and are inside GOPATH `GO111MODULE` needs to be set to `GO111MODULE=on`.
+## Developing Locally
 
-Download and prepare Prebid Server:
+Prebid Server requires [Go](https://go.dev) version 1.19 or newer. You can develop on any operating system that Go supports; however, please note that our helper scripts are written in bash.
 
-```bash
-cd YOUR_DIRECTORY
-git clone https://github.com/prebid/prebid-server src/github.com/prebid/prebid-server
-cd src/github.com/prebid/prebid-server
+1. Clone The Repository
+``` bash
+git clone git@github.com:prebid/prebid-server.git
+cd prebid-server
 ```
 
-Run the automated tests:
+3. Download Dependencies
+``` bash
+go mod download
+```
 
+3. Verify Tests Pass
 ```bash
 ./validate.sh
 ```
 
-Or just run the server locally:
-
+4. Run The Server
 ```bash
-go build .
-./prebid-server
+go run .
 ```
 
-Run format:
-```
-make format
-```
-or
-```bash
-./scripts/format.sh -f true
-```
```

Comment:
Do you think it makes sense to leave it here?

Reply:
Added back via a "code style" section.